tutorials/training.ipynb

```diff
@@ -436,7 +432,11 @@
       "source": [
         "The dataframes contain `phase`, `epoch`, `entry`, `output`, `target`, and `loss` columns, and can be easily used to visualize the results.\n",
         "\n",
-        "For example, the loss across the epochs can be plotted for the training and the validation sets:\n"
+        "For classification tasks, as in the current tutorial, the `output` column contains the results from a [softmax function](https://pytorch.org/docs/stable/generated/torch.nn.functional.softmax.html). This means each entry in the `output` column is a list with one element for each class. Each element represents the probability of that class occurring.\n",
+        "\n",
+        "Here specifically, the `output` column contains a list with two elements, respectively representing the predicted probabilities that the data point is 0 (first element of the list, representing non-binder class) and 1 (second element of the list, representing binder class).\n",
```

Comment:
why is 0 first and 1 second and not the other way around?

Reply:
They are ordered according to the `classes` attribute. I added a comment about that :)